database/plugin/metadata/sqlite/transaction.go

```diff
@@ -123,3 +127,24 @@ func (d *MetadataStoreSqlite) SetTransaction(
 	}
 	return nil
 }
+
+// createTransactionMetadata creates metadata for a transaction
+func createTransactionMetadata(tx lcommon.Transaction, point ocommon.Point, idx uint32) []byte {
```

Comment:
I don't think this is quite what we had in mind with the ticket, but that's probably on us 😅 

Each transaction can contain metadata, which is an arbitrary data structure that provides additional context for a transaction. This structure is used for things like notes about the TX or even NFT metadata.

This metadata (if any) for a TX is available via the `.Metadata()` function on the `ledger.Transaction` object, which returns a `*cbor.LazyValue`. You can call `.Cbor()` on that to get the original metadata bytes to store in the databaes.